scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -1429,6 +1431,65 @@ while true; do
+      SERVICE_EXISTS=$(systemctl list-units --type=service --all | grep -q 'cnode-mithril-signer.service'; echo $?)
+      SIGNER_SERVICE_STATUS=$(systemctl status cnode-mithril-signer.service 2>/dev/null | grep 'Active:' | awk '{print $3}' | sed 's/[()]//g')
+      MITHRIL_SIGNER_METRICS=$(curl -s http://127.0.0.1:9090/metrics 2>/dev/null | grep -v -E "HELP|TYPE" | sed 's/mithril_signer_//g')
+
+      if [[ "$nodemode" = "Core" ]] && [[ "$SERVICE_EXISTS" -eq 0 ]] && [[ "$SIGNER_SERVICE_STATUS" == "running" ]]; then
+
+        printf "${mithrildivider}\n" && ((line++))
+
+        get_metric_value() {
+            local metric_name="$1"
+            echo "$MITHRIL_SIGNER_METRICS" | awk -v metric="$metric_name" '{for(i=1;i<=NF;i++){if($i==metric){print $(i+1)}}}'
+        }
+
+        metrics=(
+            "runtime_cycle_total_since_startup"
+            "signer_registration_success_last_epoch"
+            "signer_registration_success_since_startup"
+            "signer_registration_total_since_startup"
+            "signature_registration_success_last_epoch"
+            "signature_registration_success_since_startup"
+            "signature_registration_total_since_startup"
+        )
+
+        cycle_total_VAL=$(get_metric_value "runtime_cycle_total_since_startup")
+        signer_reg_epoch_VAL=$(get_metric_value "signer_registration_success_last_epoch")
+        signer_reg_success_VAL=$(get_metric_value "signer_registration_success_since_startup")
+        signer_reg_total_VAL=$(get_metric_value "signer_registration_total_since_startup")
+        signatures_epoch_VAL=$(get_metric_value "signature_registration_success_last_epoch")
+        signatures_reg_success_VAL=$(get_metric_value "signature_registration_success_since_startup")
+        signatures_reg_total_VAL=$(get_metric_value "signature_registration_total_since_startup")
+
+        if [[ ${VERBOSE} = "Y" ]]; then
+          printf "${VL} Status     : ${style_values_2}%-${three_col_1_value_width}s${NC}" "$SIGNER_SERVICE_STATUS"
+          printf "           : Registered Epoch     : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$signer_reg_epoch_VAL"
+          closeRow
+          printf "${VL} Registered : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$signer_reg_success_VAL"
+          printf "           : Registered Total     : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$signer_reg_total_VAL"
+          closeRow
+          printf "${VL} Cycles     : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$cycle_total_VAL"
+          printf "           : Signing in Epoch     : ${style_values_2}%-${three_col_1_value_width}s${NC}" "$signatures_epoch_VAL"
+          closeRow
+          printf "${VL} Signatures : ${style_values_2}%-${three_col_1_value_width}s${NC}" "$signatures_reg_success_VAL"
+          printf "           : Total Signatures     : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$signatures_reg_total_VAL"
+          closeRow
+        else
```

Comment:
Moving the Registed / Registered Total might make for a better UX experience. The user will have the same field order in Verbose non Verbose modes, instead of inserting them between the Status and Cycles rows.
```suggestion
        if [[ ${VERBOSE} = "Y" ]]; then
          printf "${VL} Status     : ${style_values_2}%-${three_col_1_value_width}s${NC}" "$SIGNER_SERVICE_STATUS"
          printf "           : Registered Epoch     : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$signer_reg_epoch_VAL"
          closeRow
          printf "${VL} Cycles     : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$cycle_total_VAL"
          printf "           : Signing in Epoch     : ${style_values_2}%-${three_col_1_value_width}s${NC}" "$signatures_epoch_VAL"
          closeRow
          printf "${VL} Signatures : ${style_values_2}%-${three_col_1_value_width}s${NC}" "$signatures_reg_success_VAL"
          printf "           : Total Signatures     : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$signatures_reg_total_VAL"
          closeRow
          printf "${VL} Registered : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$signer_reg_success_VAL"
          printf "           : Registered Total     : ${style_values_1}%-${three_col_1_value_width}s${NC}" "$signer_reg_total_VAL"
          closeRow
        else
```